### PR TITLE
[FIX] l10n_de: tax report input positive

### PR DESCRIPTION
**Pull request by rugo-odoo**
In the German tax report, they like to have all their taxes express in positive number. This PR changes the symbol of deductible taxes amount from negative to positive.

task-3100357

---
I confirm I have signed the CLA and read the PR guidelines at www.odoo.com/submit-pr

### CODE REVIEW

File: addons/l10n_de/data/template/account.tax-de_skr04.csv
[the remark this answers is not in the document]
Comment (rugo-odoo) [reply addons/l10n_de/data/template/account.tax-de_skr04.csv:1]: @tsb-odoo

**Comment by rugo-odoo** [comment]
@Levizar can you take another look please ?

**Review by Levizar — CHANGES_REQUESTED** [review]
Regarding the content of the code: Good job. :100: 

Regarding the commit messages, those are a bit fucked up: :monocle_face:
- first commit message is wrong: it says it is about the sign of the tags but it is just about its name.
- second commit message is correct regarding what it does but has its message displayed twice

**Review by Levizar — CHANGES_REQUESTED** [review]
https://github.com/odoo/odoo/pull/146317#pullrequestreview-2309372176

**Review by Levizar — APPROVED** [review]
@robodoo r+

**Comment by Levizar** [comment]
@robodoo r-

**Comment by Levizar** [comment]
@robodoo r+

**Comment by Levizar** [comment]
@robodoo r+

**Pull request closed without merging**